packages/nc-gui/components/account/Token.vue

```diff
@@ -75,17 +90,19 @@ const deleteToken = async (token: string): Promise<void> => {
 }
 
 const generateToken = async () => {
+  isTokenNameEmpty.value = !selectedTokenData.value.description?.length || selectedTokenData.value.description?.length > 255
+
+  if (isTokenNameEmpty.value) return
   try {
     await api.orgTokens.create(selectedTokenData.value)
     showNewTokenModal.value = false
     // Token generated successfully
     // message.success(t('msg.success.tokenGenerated'))
     selectedTokenData.value = {}
     await loadTokens()
-  } catch (e: any) {
-    message.error(await extractSdkResponseErrorMsg(e))
+  } finally {
+    $e('a:api-token:generate')
```

Comment:
By removing catch block, errors thrown from `await api.orgTokens.create(selectedTokenData.value)` won't be caught.